airbyte-api/src/main/openapi/config.yaml

```diff
@@ -4729,6 +4782,31 @@ components:
           $ref: "#/components/schemas/ResourceId"
         workspaceId:
           $ref: "#/components/schemas/WorkspaceId"
+    AttemptId:
+      type: integer
```

Comment:
nit: we already have attempt id defined [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-api/src/main/openapi/config.yaml#L3936). Although attempt id should never be get to an int64, I think we shoudl standardise the types.

Reply:
@xiaohansong fyi